components/developer/byacc/test/results-64.master

```diff
@@ -0,0 +1,436 @@
+** making check
+** Thu Jan  9 18:55:26 CET 2025
+** testing flags help -z
```

Comment:
You should add a COMPONENT_TEST_TRANSFORMS line to the Makefile in order to get rid of the date and time here. Otherwise every gmake test will fail because of the changed time and date.